test-apps/display-test-app/debug.log

```diff
@@ -0,0 +1 @@
+[0105/143329.251:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
This file shouldn't have been committed.

Reply:
Fixed